yang/vyatta-op-protocols-frr-isis-v1.yang

```diff
@@ -0,0 +1,295 @@
+module vyatta-op-protocols-frr-isis-v1 {
+	namespace "urn:vyatta.com:mgmt:vyatta-op-protocols-frr-isis:1";
+	prefix vyatta-op-protocols-frr-isis-v1;
+
+	import vyatta-types-v1 {
+		prefix types;
+	}
```

Comment:
This import can be removed as it is unused. Removing it will suppress a warning from pyang:
```
vyatta-op-protocols-frr-isis-v1.yang:5: warning: imported module vyatta-types-v1 not used
```

Reply:
This is Resolved.